### ISSUE TITLE
[MDEP-320] Maven Dependency Plugin should provide an option to download components in parallel

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-dependency-plugin#763]
**[Sidharth Gupta](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=sid)** opened **[MDEP-320](https://issues.apache.org/jira/browse/MDEP-320?redirect=false)** and commented

In an environment constrained by network speed, or for a server which allocates lower priority to nexus; the download speed is quite slow. This can be quicker by starting the download of artifacts in parallel. I have written a plugin (which provides a goal) to achieve this functionality and I wish to know if I can contribute it for adoption and if so, how?

---
No further details from [MDEP-320](https://issues.apache.org/jira/browse/MDEP-320?redirect=false)

**Comment by jira-importer** [comment]
**[Michael Osipov](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=michael-o)** commented

This issue has been auto closed because it has been inactive for a long period of time. If you think this issue still persists, retest your problem with the most recent version of Maven and the affected component, reopen and post your results.
